generators/client/templates/angular/src/main/webapp/app/_app.module.ts

```diff
@@ -22,6 +22,7 @@ import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { Ng2Webstorage } from 'ng2-webstorage';
 
+import { AppRoutingModule} from './app-routing.module';
```

Comment:
Use <%=angularXAppName%> instead of App

Reply:
Ill merge this once this change is done

Reply:
Not sure it brings values `AppRoutingModule`  is an application level module.  We should prefix <%=angularXAppName%> feature modules not top level module.

Reply:
Its more about consistency as we preprnd the top level app module with it as well

Reply:
ok will do